### PR TITLE
BZ:2068127 - Updating kata config YAML files to include kataMonitorIm…

### PR DESCRIPTION
**Pull request by MirzWeiss**
Update to main and 4.10 for [BZ-2068127](https://bugzilla.redhat.com/show_bug.cgi?id=2068127)

[Preview link for web console procedure update](https://deploy-preview-44001--osdocs.netlify.app/openshift-enterprise/latest/sandboxed_containers/deploying-sandboxed-container-workloads#sandboxed-containers-create-kataconfig-resource-web-console_deploying-sandboxed-containers)
 
[Preview link for CLI procedure update](https://deploy-preview-44001--osdocs.netlify.app/openshift-enterprise/latest/sandboxed_containers/deploying-sandboxed-container-workloads#sandboxed-containers-create-kataconfig-rsource-cli_deploying-sandboxed-containers) 

Acked by SME and QE.

### CODE REVIEW

File: modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc [hunk 1]
```diff
@@ -50,9 +52,10 @@ If you want to install `kata` as a `RuntimeClass` only on selected nodes, includ
  metadata:
    name: cluster-kataconfig
  spec:
-   kataConfigPoolSelector:
-     matchLabels:
-     <label_key>: '<label_value>'
+  kataMonitorImage: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:1.2.0
```
[the remark this answers is not in the document]
Comment (MirzWeiss) [reply modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc:55]: I used the updated yaml in the bug:

```
apiVersion: kataconfiguration.openshift.io/v1
kind: KataConfig
metadata:
  name: cluster-kataconfig
spec:
  kataMonitorImage: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel8:1.2.0
  kataConfigPoolSelector:
    matchLabels:
    <label_key>: '<label_value>'
```


There are two indents before kataMonitorImage.

[the remark this answers is not in the document]
Comment (MirzWeiss) [reply modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc:55]: @jensfr ?

**Review by cpmeadors — APPROVED** [review]
Changes look good.  Assuming Jens's comment about indentation is resolve, I approve.

File: modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (jensfr) [reply modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc:55]: @MirzWeiss there is a leading space at the beginning of every line that should be removed. And <label_key> needs to be indented by two more spaces. With these changes applying the yaml file works and the operator also shows the correct behaviour.

**Review by jensfr — APPROVED** [review]
looks good, thank you @MirzWeiss

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

**Comment by bergerhoffer** [comment]
I labeled this for 4.11 as well, let me know if this is okay (and if so, I'll merge!)

**Pull request merged**